README.md

```diff
@@ -36,17 +36,20 @@ var importer = rouxSassImporter(sass.NULL, {
   pantrySearchPaths: [
     path.resolve('node_modules'),
     path.resolve('some/other/path')
-  ];
+  ]
```

Comment:
is there a code edit that needs to be done to match this, or was this just a mistake in the docs?